polkadot-parachains/src/cli.rs

```diff
@@ -1,8 +1,7 @@
-// Copyright 2021 Integritee AG and Supercomputing Systems AG
-// This file is part of the "Integritee parachain" and is
-// based on Cumulus from Parity Technologies (UK) Ltd.
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
 
-// Integritee parachain is free software: you can redistribute it and/or modify
+// Cumulus is free software: you can redistribute it and/or modify
```

Comment:
Makes sense to me, still wondering if this is intentional?

Reply:
not intentional. but I'll leave it as it's copy-paste